bundles/org.openhab.core.compat1x/src/main/java/org/openhab/io/net/http/HttpUtil.java

```diff
@@ -186,22 +186,22 @@ public static String executeUrl(String httpMethod, String url, Properties httpHe
 
         if (logger.isDebugEnabled()) {
             try {
-                logger.debug("About to execute '" + method.getURI().toString() + "'");
+                logger.debug("About to execute '{}'", method.getURI().toString());
```

Comment:
Minor you don't have to call toString it will be called automatically, but only when the logging-level actually applies.